tests/specs/issue_fix/35_resolve_todo_re_check_options_route_in_fractal_action/mysql/config/urls.rest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * OpenAPI UrlRules
+ *
+ * This file is auto generated.
+ */
+return [
+    'GET api/v1/pets' => 'api/v1/pet/list',
+    'POST api/v1/pets' => 'api/v1/pet/create',
+    'GET animals/pets/<id:[\w-]+>' => 'pet/view',
+    'DELETE animals/pets/<id:[\w-]+>' => 'pet/delete',
+    'PATCH animals/pets/<id:[\w-]+>' => 'pet/update',
+    'GET petComments' => 'pet-comment/list',
+    'GET info/pet-details' => 'petinfo/pet-detail/list',
+    'GET forum/pet2-details' => 'forum/pet2-detail/list',
+    'GET forum2/pet3-details' => 'forum2/pet3-detail/list',
+    'GET api/v2/comments' => 'api/v2/comment/list',
+    'api/v1/pets' => 'some/pet/options',
+    'animals/pets/<id:[\w-]+>' => 'pet/options',
+    'petComments' => 'pet-comment/options',
+    'info/pet-details' => 'petinfo/pet-detail/options',
+    'forum/pet2-details' => 'forum/pet2-detail/options',
+    'forum2/pet3-details' => 'forum2/pet3-detail/options',
+    'api/v2/comments' => 'api/v2/comment/options',
```

Comment:
why do auto generated rules contain module paths? I'd expect only controller/action here

Reply:
That is due to `urlPrefixes` https://github.com/php-openapi/yii2-openapi/blob/592b309659a22ce958a0b83598f3f4d29085943a/src/generator/ApiGenerator.php#L77

It is configured here in https://github.com/php-openapi/yii2-openapi/blob/3a44f9bb3c912739f72385cc44ade67e7aa3e77f/tests/specs/issue_fix/35_resolve_todo_re_check_options_route_in_fractal_action/index.php#L13